content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -6,6 +6,35 @@ keywords: [locality,load balancing,priority,prioritized,kubernetes,multicluster]
 test: yes
 owner: istio/wg-networking-maintainers
 ---
+
+{{< tip >}}
```

Comment:
one more nit 😄 I think it would be better to put this tip just before the `Environment Variables` section starts (i.e., after the setup diagram)

Reply:
I moved the tip before the `Environment Variables` section (after the setup diagram).